Source/Shaders/ModelExperimental/FeatureStageFS.glsl

```diff
@@ -0,0 +1,25 @@
+float featureId;
+vec2 featureSt;
```

Comment:
Could we have `featureStage()` write to an `inout` struct like I do with the processed attributes in my branch? This way we have something to pass around to other things (eventually custom shaders and styling)?

 I want to avoid global variables, as it makes it harder to move around pipeline stage shader code if needed.

Reply:
also (not now but in the future) would it make sense to store things like this?

```glsl
struct FeatureIdentification {
    float featureId_0; // or this could be an array
    float featureId_2;
    vec2 featureSt_0;
    vec2 featureSt_1;
};
```

Just a thought

Reply:
Based on offline discussion, @ptrgags and I agreed that this is best addressed in the CPU styling PR that will follow this one.

Reply:
We talked about this in Slack, we think this is better handled in a future PR once we have some new `ShaderBuilder` features from #9806